i18n/create_msg.js

```diff
@@ -63,6 +77,19 @@ Blockly.CatblocksMsgs.getCurrentLocaleValues = function() {
   return Blockly.CatblocksMsgs.locales[Blockly.CatblocksMsgs.getCurrentLocale()];
 };
 
+Blockly.CatblocksMsgs.loadNewLocale = function(locale) {
+  const scripts = document.getElementsByTagName("script");
+  const baseURI = scripts[0].baseURI;
```

Comment:
Very unreliable method, if we would use a script from a CDN first f.e.
Consider using window.location.host / window.location.protocol instead

Reply:
changed it, thanks for the tip